website/src/components/errors/ErrorAvatarLoading.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+import "./errors.css";
+import errorImage from "../../images/errors/error-robotmobile.png";
+
+export const ErrorAvatarLoading: React.FC = () => (
+  <div id="error-avatarerror" className="error-container">
+    <div className="error-background" />
+    <div className="errormessage">
+      <div className="errortext col">
+        <div className="error">Oops...</div>
+        <div className="communicationslink">
+          There was a problem loading your avatar
+        </div>
+        <div className="givesomedetailof">
+          It seems that the body shape of your avatar could not be loaded correctly.
+          <br />
+          <br />
+          Please try again later, or reach out to us at{" "}
```

Comment:
In order to prevent support pressure, we could add something like "If the problem persists, please reach out to us..."

Reply:
Same as before... I'm currently confirming with @olavra the definitive messages for the user feedback.

Reply:
Messages already confirmed for this error page:

**Title:** "_There was a technical issue and we were unable to retrieve your avatar information_"
**Subtitle:** "_Please try again later, and if the problem persists you can contact us through the Discord channel or at hello@decentraland.org_"

Done in https://github.com/decentraland/explorer/pull/2300/commits/2f5e380c514269c1f41758d4e8342a6f4da7f3b2